dev-docs/module-rules.md

```diff
@@ -40,65 +40,77 @@ Bid adapters, user ID adapters, analytics adapters, and other modules will be re
 1. All modules must support the creation of multiple concurrent instances. This means, for example, they cannot rely on mutable global variables.
-    - Writing to the global window object must be non-intrusive, disclosed, and done in a way that supports multiple instances.
-1. All modules and any external code references must disclose their support of privacy regulations such as GDPR, CCPA, COPPA, etc. (See the disclosure section below.)
+    - PBJS: Writing to the global window object must be non-intrusive, disclosed, and done in a way that supports multiple instances.
+1. Modules must not include polyfills or override standard or Prebid JavaScript functions.
+1. Endpoint domain names cannot be fully variable. At some point, they won't be able contain any variables at all. e.g. $PARAM1.example.com.
+1. In order to be listed on the Prebid.org website, a module must be in a Prebid open source repository.
+1. Prebid.js requests and responses must be secure HTTPS.
+1. Prebid.js modules must not use the PREBID_GLOBAL variable nor otherwise obtain a pointer to the global PBJS object. This preserves data integrity.
+
+#### Rules that May Someday Be Enforced
+
 1. Responses to auction or data requests should be compressed as long as the client supports that functionality (e.g. gzip, compress, deflate).
-1. Prebid.js requests and responses must be secure HTTPS. Prebid Server requests should be HTTPS.
 1. Endpoints should use HTTP/2 with keepalives so that connections don’t need to be re-created.
-1. In order to be listed on the Prebid.org website, a module must be in a Prebid open source repository.
-1. Each module must include maintainer information with a group email address for Prebid.org to contact for support or maintenance.
-1. Modules must not include polyfills or override standard or Prebid JavaScript functions.
 1. External modules must not use getEidPermissions function of userId module (e.g. prebidServerBidAdapter is an internal prebid module). 
+1. All modules and any external code references must disclose their support of privacy regulations such as GDPR, CCPA, COPPA, etc. (See the [disclosure](#disclosure) section below.)
 
 ### Bidder Adapter Rules
 
-1. All global rules apply.
+All global rules apply.
+
+#### Stricly Enforced Rules
+
 1. Bid adapters must be able to bid. If they cannot return an auction bid, they should consider integration as a Real-Time Data or other module type.
-1. Creative rendering scripts must(*) load from the creative frame and not directly in the page. In other words, creative rendering JavaScript cannot be used to bypass the “no external code” rule.
-1. Bidder modules must not(*) make requests to endpoints for functionality other than auctions without:
+1. Bidder modules must not make requests to endpoints for functionality other than auctions without:
```

Comment:
Would this be PBJS specific? For server, issue https://github.com/prebid/prebid-server/issues/2437 states "Bidder modules must not make requests to endpoints for functionality other than auctions." without exception.

Reply:
The rules committee allowed for the possibility of exceptions for JS -- I think that would apply to PBS as well